app/models/miq_report/import_export.rb

```diff
@@ -27,7 +27,6 @@ def resolve_view_path(file_name, file_name_no_suffix = nil)
     def import_from_hash(report, options = nil)
       raise _("No Report to Import") if report.nil?
 
-      report = report["MiqReport"] if report.keys.first == "MiqReport"
```

Comment:
this is here to support importing a yaml file with multiple reports in them. those seem to have the extra `MiqReport` nodes.